test/unit/cpu/CMakeLists.txt

```diff
@@ -41,7 +41,9 @@
 #
 ###############################################################################
 
+set_source_files_properties(buildIndexSet.cpp PROPERTIES CUDA_SOURCE_PROPERTY_FORMAT OBJ)
```

Comment:
This should be automatically handled by the `raja_add_library` macro - what was happening?